coralproject/talk
# Fix firefox modal bug (Issue 148)

## What does this PR do?

Fixes a bug where comments cannot be entered on Firefox after logging in. (https://github.com/coralproject/talk/issues/148)

## How do I test this PR?

1) Log in using firefox.
2) Click the comment box. You should be able to enter text.
